iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSplit.java

```diff
@@ -75,7 +81,7 @@ public Path getPath() {
 
   @Override
   public byte[] getBytesForHash() {
-    Collection<FileScanTask> fileScanTasks = innerSplit.task().files();
+    Collection<FileScanTask> fileScanTasks = innerSplit.task().tasks();
```

Comment:
could we remove the intermediate `CombinedScanTask` and return `FileScanTask`s right away, I don't see where we use `CombinedScanTask` directly
````
innerSplit.tasks()
````